src/pages/username-select/index.tsx

```diff
@@ -29,15 +29,18 @@ const styles = {
 const UsernameSelect: FC = () => {
   const router = useRouter();
   const { data: session } = useSession();
-  const { address, getSignature } = useBBWallet();
+  const { connect, isConnected, address, getSignature } = useBBWallet();
 
   const verify = api.user.verify.useMutation();
-
   useEffect(() => {
     if (isVerifiedSession(session)) {
       void router.push(pages.home);
     }
-  }, [router, session]);
+
+    if (!isConnected) {
+      void connect();
+    }
```

Comment:
Why this? Isn't user supposed to be already connected to the wallet when here?

Reply:
No, When the page is reloaded the connection to the wallet is gone. public key is. After reloading the publicKey is undefined.